cpp/JsiWorkletApi.h

```diff
@@ -130,8 +130,17 @@ class JsiWorkletApi : public JsiHostObject {
         runtime, JsiWorkletContext::getDefaultInstance());
   }
 
+  JSI_PROPERTY_GET(currentContext) {
+    auto current = JsiWorkletContext::getCurrent(runtime);
+    if (!current) return jsi::Value::undefined();
+    return jsi::Object::createFromHostObject(
+        runtime, current->getptr());
```

Comment:
No need to add this method, JsiWorkletContext has this:

```suggestion
    return jsi::Object::createFromHostObject(
        runtime, current->shared_from_this());
```